packages/indexer-client/src/types/IndexerLeaderboardType.ts

```diff
@@ -0,0 +1 @@
+export type IndexerLeaderboardRankType = 'pnl' | 'roi';
```

Comment:
Can we leave a comment explaining the difference? pnl is absolute pnl, roi is %

Reply:
Yup, will do

Reply:
addressed!